projects/plugins/jetpack/extensions/store/membership-products/resolvers.js

```diff
@@ -151,13 +151,15 @@ const setDefaultProductIfNeeded = ( selectedProductId, setSelectedProductId, sel
 	}
 };
 
-export const getNewsletterProducts = (
+export const getNewsletterTierProducts = (
 	productType = PRODUCT_TYPE_PAYMENT_PLAN,
 	selectedProductId = 0,
 	setSelectedProductId = () => {}
 ) =>
 	// Returns the products, but silences the snack bar if a default product is created
-	getProducts( productType, selectedProductId, setSelectedProductId, false );
+	getProducts( productType, selectedProductId, setSelectedProductId, false ).filter(
+		product => product.type === TYPE_TIER
```

Comment:
they will always have a `type`?

Reply:
Tiers? Yes. It is explained p81Rsd-1wJ-p2